src/solver/variable/info.h

```diff
@@ -407,7 +407,7 @@ struct VariableAccessor<ResultsT, Category::dynamicColumns>
         if (renewable_details)
         {
             auto& renewable = results.data.area->renewable;
-            results.variableCaption = renewable.clusters[idx]->name();
+            results.variableCaption = renewable.list.byIndex[idx]->name();
```

Comment:
**Question** (the question is at the end) : 
Here, as a replacement for `renewable.clusters`, we use `renewable.list.byIndex`.
But somewhere else (for instance in **timeseries-numbers.cpp**), we replace a loop over `renewable.clusters` with a loop over `renewable.list` (which actually means looping over `renewable.list.cluster`).

It could mean that  `renewable.list.cluster` and `renewable.list.byIndex` are the more or less the same and that one them is redundant. 
**What do you think ?**